quic/s2n-quic-core/src/recovery/bbr/snapshots/quic__s2n-quic-core__src__recovery__bbr__probe_bw__events__enter_probe_bw.snap

```diff
@@ -0,0 +1,7 @@
+---
+source: quic/s2n-quic-core/src/recovery/bbr/probe_bw.rs
+expression: ""
+---
+BbrStateChanged { path_id: 0, state: ProbeBwDown }
```

Comment:
Should we only emit an event if it's actually changed?

Reply:
This test is manually overriding the state to Drain before called enter_probe_bw, so it actually is changing. The `transition_to` methods enforce that the state/cycle phase is actually changing